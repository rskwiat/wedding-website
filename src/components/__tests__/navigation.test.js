import React from 'react';
import { shallow } from 'enzyme';
import Navigation from '../Navigation';

test('Application should mount', () => {
  const component = shallow(<Navigation />);
  expect(component).toMatchSnapshot();
})