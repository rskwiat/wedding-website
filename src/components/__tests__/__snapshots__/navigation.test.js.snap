// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application should mount 1`] = `
<nav>
  <ul
    className="desktopNav"
  >
    <NavLink
      key="0"
      title="Home"
      url="/"
    />
    <NavLink
      key="1"
      title="Wedding"
      url="/wedding"
    />
    <NavLink
      key="2"
      title="RSVP"
      url="/rsvp"
    />
    <NavLink
      key="3"
      title="Registry"
      url="/registry"
    />
  </ul>
</nav>
`;
